sample.env

```diff
@@ -14,10 +14,14 @@ RPC_URL=
 KAFKA_CLIENT_ID=
 
 # Kafka topic
-KAFKA_TOPIC=defender
+BROKER_TOPIC=defender
 
-# Pubsub (default: kafka)
-PUBSUB_SERVICE=kafka
+# Pubsub (default: rabbitmq)
+PUBSUB_SERVICE=rabbitmq
+
+RABBITMQ_URL=wss://dsm.testnet.fi/ws
+RABBITMQ_LOGIN=test
+RABBITMQ_PASSCODE=test
```

Comment:
I can connect with these or any other credentials and get no error. Are we disabling auth on the server or are we not handling auth errors?

Reply:
Fixed